packages/flame/lib/src/extensions/vector2.dart

```diff
@@ -123,12 +123,21 @@ extension Vector2Extension on Vector2 {
   ///
   /// Note: [ds] is the displacement vector in units of the vector space. It is
   /// **not** a percentage (relative value).
+  ///
+  /// The [efficientDiff] is an optional parameter that can be used to avoid
+  /// creating a new [Vector2] when calling this method multiple times.
   void moveToTarget(
     Vector2 target,
-    double ds,
-  ) {
+    double ds, {
+    Vector2? efficientDiff,
```

Comment:
Maybe we could just have a private static vector in here instead so that the user doesn't have to pass in this vector?

Reply:
That's a great idea!
What do you think about sharing it inside the extension by naming it generally?
It might be useful when there is some point to save performance same as `moveToTarget`.

Reply:
Sounds good :)